library/certs.c

```diff
@@ -116,31 +116,6 @@ const size_t mbedtls_test_cli_key_ec_len = sizeof( mbedtls_test_cli_key_ec );
 #endif /* MBEDTLS_ECDSA_C */
 
 #if defined(MBEDTLS_RSA_C)
-#if defined(MBEDTLS_SHA1_C)
```

Comment:
As discussed in #978, I don't think we need to make this change. However, I'm willing to go along with it given you've already made the change in the PR.